source/server/getting-started/migrating.rst

```diff
@@ -29,19 +32,42 @@ Worlds
 
 Forge, and thus SpongeForge (and also SpongeVanilla), use the same world structure as vanilla Minecraft. Vanilla Minecraft
 places the nether (typically ``world_nether``) and the end (typically ``world_the_end``) dimensions within the
-``world`` folder.
+``world`` folder. However Bukkit and Spigot *don't* use this system to save the worlds, thus migration is needed.
+
+SpongeForge and SpongeVanilla provide a fully automated conversion script which converts your worlds for you.
+This is how it works:
+
+1. Shutdown your Bukkit or Spigot server and Backup the entire folder.
+
+#. Place SpongeForge or SpongeVanilla in the folder where you ran your old server, remove the Bukkit or Spigot jars.
```

Comment:
Maybe link to the docs about setting up a SpongeForge/SpongeVanilla server here.